File: AlphaParking.Java.AuthService/javaauth/src/main/java/com/auth/event_bus_utils/integration_events/UserRemovedIntegrationEvent.java
package com.auth.event_bus_utils.integration_events;

import com.auth.models.User;

public class UserRemovedIntegrationEvent extends IntegrationEvent{
    private User user;

    public UserRemovedIntegrationEvent(User user) {
        this.setUser(user);
    }

    /**
     * @return the user
     */
    public User getUser() {
        return user;
    }

    /**
     * @param user the user to set
     */
    public void setUser(User user) {
        this.user = user;
    }

}

File: AlphaParking/AlphaParking.BLL/EventBus/IntegrationEvents/UserRemovedIntegrationEvent.cs
using AlphaParking.BLL.EventBus.Abstractions;
using AlphaParking.Models;

namespace AlphaParking.BLL.EventBus.IntegrationEvents
{
    public class UserRemovedIntegrationEvent: IntegrationEvent
    {
        public User User { get; set; }

        public UserRemovedIntegrationEvent(User user)
            => User = user;
    }
}